app/mailers/candidate_mailer.rb

```diff
@@ -509,15 +509,35 @@ def candidate_invite(pool_invite)
     @pool_invite = pool_invite
     @preferences_url = candidate_preferences_link(pool_invite.candidate)
     @invite_url = edit_candidate_interface_invite_url(pool_invite)
-    application_form = pool_invite.application_form
+    @application_form = pool_invite.application_form
+    @not_responded_invites = @application_form.not_responded_published_invites.count
```

Comment:
Not a big deal, but could we change this to `@not_responded_invites_count` -- i tripped over it when we call `TextOrdinalizer`, thinking you were passing a collection of invites rather than a count.